### PR TITLE
Fix aurora mysql 5.6 issue

### PR DESCRIPTION
**Pull request by nappelson**
Can see results with db 125 in nappelson@ottertune.com organization

### CODE REVIEW

**Comment by nappelson** [comment]
@YuzeLiao sorry this includes some formatting changes that happened

File: driver/driver_config_builder.py [hunk 1]
```diff
@@ -256,10 +257,8 @@ def _get_cloudwatch_metrics_file(self, db_instance_identifier):
         db_type = get_db_type(db_instance_identifier, self.rds_client)
 
         if "aurora" in db_type:
-            if db_type == "aurora":
-                db_type = "aurora_mysql"
-
-            db_version_breakdown = db_version.split("_")
+            if db_type == "aurora_mysql":
+                db_version_breakdown = db_version.split("_")
```
Comment (YuzeLiao) [inline driver/driver_config_builder.py:261]: maybe leave this outside? otherwise postgresql cannot find the variable `db_version_breakdown`

**Review by YuzeLiao — APPROVED** [review]
One minor issue the others LGTM

File: driver/driver_config_builder.py (hunk 1 above)
Comment (nappelson) [reply driver/driver_config_builder.py]: thanks! Seeing that the tests caught this too. yay tests!

**Pull request merged**